website/docs/attached-class.md

```diff
@@ -0,0 +1,75 @@
+---
+id: attached-class
+title: T.attached_class
+---
+
+`T.attached_class` can be used to refer to the type of instances from a
+singleton-context, in a way that is friendly to inheritance.
+
+```ruby
+class AbstractData
+  extend T::Sig
+
+  sig {params(id: Integer).returns(T.attached_class)}
+  def self.load(id)
+    new
+  end
+end
+
+class Article < AbstractData; end
+class User < AbstractData; end
+class Administrator < User; end
+
+T.reveal_type(Article.load(10)) # Article
+T.reveal_type(Administrator.load(20)) # Administrator
```

Comment:
As a reader I was expecting to see 3 lines here to match the structure from above, and when they weren't all there I back solved for which one had been omitted. Not sure if that's the effect you were going for or not.